udata/migrations/2020-08-24-add-fs-filename.py

```diff
@@ -32,13 +35,14 @@ def migrate(db):
 
     community_resources = CommunityResource.objects().no_cache().timeout(False)
     for community_resource in community_resources:
-        parsed = urlparse(community_resource.url)
-        fs_name = parsed.path.strip('/resource/')
-        community_resource.fs_filename = fs_name
-        try:
-            community_resource.save()
-        except Exception as e:
-            log.warning(e)
-            pass
+        if community_resource.url.startswith(storages.resources.base_url):
```

Comment:
don't we need to remove `fs_filename` in an `else` because we set some we shouldn't have on the previous run?